src/donor_dashboard/views.py

```diff
@@ -230,3 +264,91 @@ def delete_donation(request, donation_id):
     return redirect(
         "donor_dashboard:manage_organization", organization_id=donation.organization_id
     )
+
+
+@login_required
+def add_org_admin(request):
+    if request.method == "POST":
+        organization_id = request.POST["organization_id"]
+        new_admin_email = request.POST.get("email")
+
+        organization = Organization.objects.get(organization_id=organization_id)
+        new_admin_user = User.objects.get(email=new_admin_email)
+        if (
+            len(
+                OrganizationAdmin.objects.filter(
+                    user=new_admin_user, organization=organization
+                )
+            )
+            == 0
+        ):
+
+            OrganizationAdmin.objects.create(
+                user=new_admin_user, organization=organization, access_level="admin"
+            )
+            messages.success(request, "Admin successfully added.")
+
+            return redirect(
+                "donor_dashboard:organization_details", organization_id=organization_id
+            )
+        else:
+            messages.success(request, "Admin already associated")
+            return redirect(
+                "donor_dashboard:organization_details", organization_id=organization_id
+            )
+
+
+@login_required
+def make_as_admin(request, organization_id, admin_email):
```

Comment:
Nit: Instead of having two functions- make_as_admin and make_as_owner, we can also have a single function maybe "assign_organization_access_role" taking the org id, email and the access level  and assign the access level to the user.

Reply:
Yes, this is a much better approach. Updated!!